src/lammpsio/box.py

```diff
@@ -116,6 +150,70 @@ def from_matrix(cls, low, matrix, force_triclinic=False):
 
         return cls(low, high, tilt)
 
+    @classmethod
+    def to_hoomd_convention(cls, box):
+        """Convert a :class:`Box` to HOOMD-blue convention.
+
+        Parameters
+        ----------
+        box : :class:`Box`
+            The box to convert.
+
+        Returns
+        -------
+        :class:`numpy.ndarray`
+            A matrix of box dimensions in HOOMD-blue convention.
+
+        """
+        if not isinstance(box, Box):
+            raise TypeError("box must be Box.")
+        L = box.high - box.low
+        if box.tilt is not None:
+            tilt = box.tilt.copy()
+            tilt[0] /= L[1]
+            tilt[1:] /= L[2]
+        else:
+            tilt = [0, 0, 0]
+
+        return numpy.concatenate((L, tilt))
+
+    @classmethod
+    def from_hoomd_convention(cls, box_data, dimensions):
```

Comment:
This should probably follow the HOOMD / freud convention by defaulting `dimensions=None`. Then:
```
if dimensions is None:
    dimensions = 3 if box_data[2] != 0 else 2

if dimensions not in (2, 3):
    raise ValueError("Dimensions must be 2 or 3")
```

Also, do we need a `force_triclinic` option here?